### PR TITLE
Add Deploy 13.2.0-rc1 and 14.1.0-rc1, Deploy Contrib 4.4.0-rc1, 10.3.0-rc1, 12.2.0-rc1, 13.2.0-rc1 and 14.1.0-rc1 release notes

### PR DESCRIPTION
**Pull request by ronaldbarendse**
## Description

Release notes for the first RC release of Deploy and Deploy Contrib minors.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Deploy 13.2.0-rc1 and 14.1.0-rc1
Deploy Contrib 4.4.0-rc1, 10.3.0-rc1, 12.2.0-rc1, 13.2.0-rc1 and 14.1.0-rc1

## Deadline (if relevant)

Friday July 19th (the releases have already been published to NuGet)

### CODE REVIEW

**Review by AndyButland — APPROVED** [review]
Release notes are fine from my site.  Looks like you just need to clean up one linting warning (using "datatype" instead of "data type").

File: 12/umbraco-deploy/release-notes.md [hunk 1]
```diff
@@ -122,7 +122,12 @@ This section contains the release notes for Umbraco Deploy 12 including all chan
 
 ## Deploy Contrib
 
-#### [12.1.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.1.0 (March 19th 2024)
+#### [12.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.2.0-rc1) (July 19th 2024)
+
+* Skip empty pre-values in [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
+* Add DocTypeGridEditor data type configuration connector in [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```
Comment (eshanrnh) [inline 12/umbraco-deploy/release-notes.md:128]:
```suggestion
* Add DocTypeGridEditor Data Type configuration connector [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```

File: 14/umbraco-deploy/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [14.1.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F14.1.0) (July 19th 2024)
+
+* Add migrators to support legacy Grid layout to Block Grid migration
+* Fix schema mismatch when saving templates in production runtime mode [#228](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/228)
+* Add default migrator to add missing editor UI aliases and update when replacing editor
+* Add default migrator to update data type configuration
```
Comment (eshanrnh) [inline 14/umbraco-deploy/release-notes.md:26]:
```suggestion
* Add default migrator to update Data Type configuration
```

File: 10/umbraco-deploy/release-notes.md [hunk 3]
```diff
@@ -482,6 +487,15 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
 <details>
 <summary>Version 4</summary>
 
+#### [4.4.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-4.4.0-rc1) (July 19th 2024)
+
+* Skip empty pre-values in [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```
Comment (eshanrnh) [inline 10/umbraco-deploy/release-notes.md:492]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```

File: 12/umbraco-deploy/release-notes.md [hunk 4]
```diff
@@ -122,7 +122,12 @@ This section contains the release notes for Umbraco Deploy 12 including all chan
 
 ## Deploy Contrib
 
-#### [12.1.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.1.0 (March 19th 2024)
+#### [12.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.2.0-rc1) (July 19th 2024)
+
+* Skip empty pre-values in [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```
Comment (eshanrnh) [inline 12/umbraco-deploy/release-notes.md:127]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```

File: 14/umbraco-deploy/release-notes.md [hunk 5]
```diff
@@ -18,6 +18,14 @@ If you are upgrading to a new major version you can find the details about the b
...
+* Support processing document/media type list view keys and add migrator
```
Comment (eshanrnh) [inline 14/umbraco-deploy/release-notes.md:27]:
```suggestion
* Support processing Document/Media Type list view keys and add migrator
```

File: 14/umbraco-deploy/release-notes.md [hunk 6]
```diff
@@ -45,6 +53,12 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
 
 ## Deploy Contrib
 
+#### [14.1.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-14.1.0-rc1) (July 19th 2024)
+
+* Add Matryoshka Group Separator artifact migrator in [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
+* Add migrators to support DocTypeGridEditor to Block Grid migration in [#66](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/66)
+* Fix and enable remaining legacy artifact migrators in [#68](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/68)
```
Comment (eshanrnh) [inline 14/umbraco-deploy/release-notes.md:60]:
```suggestion
* Add Matryoshka Group Separator artifact migrator [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
* Add migrators to support DocTypeGridEditor to Block Grid migration [#66](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/66)
* Fix and enable remaining legacy artifact migrators [#68](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/68)
```

File: 13/umbraco-deploy/release-notes.md [hunk 7]
```diff
@@ -100,7 +104,13 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
 
 ## Deploy Contrib
 
-#### [13.1.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-13.1.0 (March 19th 2024)
+#### [13.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-13.2.0-rc1) (July 19th 2024)
+
+* Skip empty pre-values in [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
+* Add Matryoshka Group Separator artifact migrator in [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
+* Add migrators to support DocTypeGridEditor to Block Grid migration in [#66](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/66)
```
Comment (eshanrnh) [inline 13/umbraco-deploy/release-notes.md:111]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
* Add Matryoshka Group Separator artifact migrator [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
* Add migrators to support DocTypeGridEditor to Block Grid migration [#66](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/66)
```

File: 10/umbraco-deploy/release-notes.md [hunk 8]
```diff
@@ -450,6 +450,11 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
 
 ## Deploy Contrib
 
+#### [10.3.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-10.3.0-rc1) (July 19th 2024)
+
+* Skip empty pre-values in [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
+* Add DocTypeGridEditor data type configuration connector in [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```
Comment (eshanrnh) [inline 10/umbraco-deploy/release-notes.md:456]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
* Add DocTypeGridEditor Data Type configuration connector [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```

**Review by eshanrnh — APPROVED** [review]
Thanks for the PR, @ronaldbarendse 💪

I've removed the word "in" to maintain consistency in the Release Notes and fixed the Data Type (vale error).

**Pull request merged**